src/state/apis/nft/nftApi.ts

```diff
@@ -129,32 +256,35 @@ export const nftApi = createApi({
 
         const results = await Promise.allSettled(services.map(service => service(accountIds)))
 
-        const dataById = results.reduce<Record<AssetId, NftItemWithCollection>>((acc, result) => {
-          if (isRejected(result)) return acc
-
-          if (result.value.data) {
-            const { data } = result.value
-
-            data.forEach(item => {
-              const { assetId } = item
-
-              if (!acc[assetId]) {
-                acc[assetId] = item
-              } else {
-                acc[assetId] = updateNftItem(acc[assetId], item)
-              }
-            })
-            // An actual RTK error, different from a rejected promise i.e getAlchemyNftData rejecting
-          } else if (result.value.isError) {
-            console.error(result.value.error)
-          }
+        const resultsDataById = results.reduce<Record<AssetId, NftItemWithCollection>>(
```

Comment:
two hard things in software engineering mang, happy to use a better name here

Reply:
`nftsWithCollectionById`